### PR TITLE
feat(lastCam): add plugin

### PR DESCRIPTION
**Pull request by Bengejd**
Added cordova-plugin-last-cam

### CODE REVIEW

**Comment by wf9a5m75** [comment]
Only iOS ...

**Comment by Bengejd** [comment]
@wf9a5m75 correct. I have not implemented an Android version of the plugin yet, but that is in the works. Is it a problem that it only targets IOS for now? I know there are a number of plugins in @ionic-native that only target one platform or the other.

**Comment by Bengejd** [comment]
That makes sense. I'll add in a disclaimer that it is only for IOS. I don't have an Android device to test on right now, so I have limited means to implement the Android version. I appreciate your feedback though.

File: src/@ionic-native/plugins/last-cam/index.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+/*
+ @Name: @ionic-native/LastCam
```
[the remark this answers is not in the document]
Comment (Bengejd) [reply src/@ionic-native/plugins/last-cam/index.ts]: Fixed.

**Pull request merged**